markdown_lab/utils/version.py

```
@@ -0,0 +1,14 @@
+"""
+Version information for the markdown_lab package.
+"""
+
+__version__ = "1.0.0"
+
+def get_version():
+    """
+    Return the current version of the markdown_lab package.
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)